File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/NovaPoshta/WarehousesResponse/Info.java

package com.artmurka.artmurkaapp.Model.Pojo.ItemList.NovaPoshta.WarehousesResponse;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Info {

    @SerializedName("totalCount")
    @Expose
    private String totalCount;

    public String getTotalCount() {
        return totalCount;
    }

    public void setTotalCount(String totalCount) {
        this.totalCount = totalCount;
    }

    public Info withTotalCount(String totalCount) {
        this.totalCount = totalCount;
        return this;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/novaposhta/WarehousesResponse/Info.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.novaposhta.WarehousesResponse

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Info {

    @SerializedName("totalCount")
    @Expose
    var totalCount: String? = null

    fun withTotalCount(totalCount: String): Info {
        this.totalCount = totalCount
        return this
    }

}
